demo/e2e/step_definitions/test_run.steps.ts

```diff
@@ -0,0 +1,38 @@
+import { defineSupportCode } from 'cucumber';
+import { browser } from 'protractor';
+import { LandingPo } from '../pages/landing.po';
+import { safeClick, waitForUrlContains, waitForVisible } from '../shared/helpers/wd-helper';
+import { GettingStartedPo } from '../pages/gettingStarted.po';
+import { BasePagePo } from '../shared/pages/basePage.po';
+
+defineSupportCode(({Given, When, Then}) => {
+  const basePage: BasePagePo = new BasePagePo();
```

Comment:
if you extend from BasePage then there is no need to create base page in each test